webui/src/pages/repositories/repository/objectViewer.tsx

```diff
@@ -103,9 +104,9 @@ const FileObjectsViewerPage = () => {
     );
 };
 
-export const FileContents: FC<FileContentsProps> = ({repoId, refId, path, loading, error, contentType = null, fileExtension='', sizeBytes = -1, showFullNavigator = true}) => {
+export const FileContents: FC<FileContentsProps> = ({repoId, refId, path, loading, error, contentType = null, fileExtension='', sizeBytes = -1, showFullNavigator = true, usePresigned= false}) => {
```

Comment:
Didn't find the FileContent used with usePresigned property

Reply:
on the objects page